.editorconfig

```diff
@@ -0,0 +1,9 @@
+# https://EditorConfig.org
```

Comment:
@Stephane-D what do you think?

This way everyone should use the same eol convention

Reply:
I think that is indeed better to have same EOL for everyone.